tests/performance/utils.js

```diff
@@ -21,15 +21,21 @@ if (global.window && global.window.location && global.window.location.search) {
 var adapterUsed;
 
 exports.runTests = function (PouchDB, suiteName, testCases, callback) {
+  if (typeof iterations === 'number') {
+    // Re-write to allow test.iterations use inside setup & tearDown.
+    // TODO a better approach would be to pass the number of iterations into
+    // setup & tearDown functions.
+    testCases.forEach((testCase) => {
+      testCase.iterations = iterations;
```

Comment:
This seems like a risky property name as it's likely to conflict but I can't figure out if you're internationally overwriting a property that's in the testCase? If not, can we call it something that's clearly bespoke?